src/home/index.html

```diff
@@ -52,5 +53,170 @@ <h1 class="heading">I am Mwafrika Glad to see you!</h1>
         </ul>
       </div>
     </section>
+    <section class="works">
+      <section class="tonic">
+        <img src="../assert/images/tonic1.png" alt="" />
+        <h1>Tonic</h1>
+        <div class="tonic-list">
+          <div class="black">CANOPY</div>
+          <div class="dot"></div>
+          <div class="gray">Back End Dev</div>
+          <div class="dot"></div>
+          <div class="gray">2015</div>
+        </div>
+        <p class="tonicDesc">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="skills">
+          <p>html</p>
+          <p>css</p>
+          <p>javascript</p>
+        </div>
+        <div class="btn">
+          <button class="btn-tonic">See Project</button>
+        </div>
+      </section>
+      <section class="tonic">
+        <img src="../assert/images/multiple.png" alt="" />
+        <h1>Multi-Post Stories</h1>
+        <div class="tonic-list">
+          <div class="black">CANOPY</div>
+          <div class="dot"></div>
+          <div class="gray">Back End Dev</div>
+          <div class="dot"></div>
+          <div class="gray">2015</div>
+        </div>
+        <p class="tonicDesc">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="skills">
+          <p>html</p>
+          <p>css</p>
+          <p>javascript</p>
+        </div>

```

Comment:
I think the list of project technologies (e.g., "html", "css", and "javascript" ) should be a list `ul` and each of them a `li`. Please add the `ul` tag instead of the `div` tag and replace the `p` tag with a `li` tag 😊 . Please follow the same approach for each section inside the "works" section.